addons/website/static/src/snippets/s_mega_menu_images_subtitles/000.scss

```diff
@@ -8,6 +8,9 @@
         &:hover, &:focus {
             background: rgba(0, 0, 0, .05);
         }
+        img {
+            align-self: start;
```

Comment:
Please move this line above `max-width: 64px` to group the rules for the same element. Thank you!
```scss
    .d-flex img {
        align-self: start;
        max-width: 64px;
    }
```